src/main/java/com/minecolonies/coremod/entity/pathfinding/pathjobs/AbstractPathJob.java

```diff
@@ -573,7 +574,7 @@ protected Path search()
             totalNodesVisited++;
 
             // Limiting max amount of nodes mapped
-            if (totalNodesVisited > MineColonies.getConfig().getServer().pathfindingMaxNodes.get() || totalNodesVisited > maxRange * maxRange)
```

Comment:
?

Reply:
the static cap doesnt really fit since depending on the job we have more or less amounts of nodes, esp the spawn pathing is rather big and would unnecessarily capped by a static amount here